Config/FilterSetCollection.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+use Liip\ImagineBundle\Imagine\Filter\FilterConfiguration;
+
+final class FilterSetCollection
+{
+    /**
+     * @var array
+     */
+    private $filterSets = [];
+
+    /**
+     * @var FilterConfiguration
+     */
+    private $filterConfiguration;
+
+    /**
+     * @var FilterSetBuilderInterface
+     */
+    private $filterSetBuilder;
+
+    /**
+     * @param FilterConfiguration       $filterConfiguration
+     * @param FilterSetBuilderInterface $filterSetBuilder
+     */
+    public function __construct(FilterConfiguration $filterConfiguration, FilterSetBuilderInterface $filterSetBuilder)
```

Comment:
does it make sense to use the FilterConfiguration object? looks to me like that is a previous attempt at this problem that did not go far enough? it smells like we should deprecate FilterConfiguration and pass the plain array here, or not?

Reply:
done